website/pages/index.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
you shouldn't need this with next

Reply:
This is so that TypeScript can parse the file with the top-level `yarn ts` typecheck command.